src/main/java/tudelft/passinggrade/PassingGrade.java

```diff
@@ -0,0 +1,32 @@
+package tudelft.passinggrade;
+
+public class PassingGrade {
```

Comment:
Can you rename this to PassingGradeConcept? Becuase I'm working on the simpler one, which literally just says whether you passed and I named it exactly the same.

Reply:
Done! Do you think we should rename the package as well?

Reply:
(and the .md as well)

Reply:
Hmm honestly, I think we could leave it in the same package because it's related.

Reply:
Ah yes, forgot the .md; Yeah, I think one package sounds good!